lifecycle-application/src/test/java/io/quarkus/ts/openshift/lifecycle/LifecycleApplicationOpenShiftIT.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.ts.openshift.lifecycle;
+
+import io.fabric8.kubernetes.api.model.PodList;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.ts.openshift.common.OpenShiftTest;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static io.restassured.RestAssured.when;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@OpenShiftTest
+public class LifecycleApplicationOpenShiftIT {
+
+    private static final String LOGGING_PROPERTY = "-Djava.util.logging.manager=org.jboss.logmanager.LogManager";
+    private static final String RUNTIME_LABEL = "app.openshift.io/runtime";
+    private static final String RUNTIME_QUARKUS = "quarkus";
+
+    @TestResource
+    private OpenShiftClient openshiftClient;
+
+    @Test
+    public void shouldArgumentsNotContainLoggingProperty() {
+        when().get("/args")
+        .then().statusCode(200)
+                .body(not(containsString(LOGGING_PROPERTY)));
+    }
+
+    @Test
+    public void shouldPrintMessagesFromQuarkusMain() {
+        PodList pods = openshiftClient.pods().withLabel(RUNTIME_LABEL, RUNTIME_QUARKUS).list();
+        assertEquals(1, pods.getItems().size(), "Found more than one pod with Quarkus runtime");
+
+        String actualLog = openshiftClient.pods().withName(pods.getItems().get(0).getMetadata().getName()).getLog();
+
+        Optional<String> argumentsLine = actualLog.lines().filter(line -> line.contains("Received arguments:")).findFirst();
+        assertTrue(argumentsLine.isPresent(), "Pod log does not contain the received arguments. Actual content: " + actualLog);
+        assertFalse(argumentsLine.get().contains(LOGGING_PROPERTY),
+                "Pod log contain wrong arguments. Actual content: " + actualLog);
```

Comment:
`argumentsLine` should be enough, instead of `actualLog`

Reply:
Good idea!